tv_apps/fling-player/test/unit/mock_presentation.js

```diff
@@ -0,0 +1,152 @@
+(function(exports) {
+  'use strict';
+
+  var proto,
```

Comment:
again, break them into several declaration sentences since you have specified initial value.